MojioSDK.xcodeproj/project.pbxproj

```diff
@@ -603,7 +621,7 @@
 				BC879B581CBEFFDD00F9177B /* RPM.swift */,
 				BC879B5A1CBEFFDD00F9177B /* Speed.swift */,
 				BC879B4A1CBEFFDD00F9177B /* TimePeriod.swift */,
-				37BC6C8223129E4B0054B420 /* EngineOilTemperature.swift */,
```

Comment:
#? intended?

Reply:
@pauluhn This is Xcode project file editing side effect. In line 614 you can see the new line for "EngineOilTemperature". So this line changed the position, it wasn't deleted